src/spark/sparkwallet.cpp

```diff
@@ -123,7 +128,10 @@ CAmount CSparkWallet::getUnconfirmedBalance() {
             continue;
 
         // Continue if confirmed
-        if (mint.nHeight > 1)
+        if (mint.nHeight > 1 && mint.type != spark::COIN_TYPE_COINBASE)
+            continue;
+
+        if (mint.type == spark::COIN_TYPE_COINBASE && (height - mint.nHeight) > COINBASE_MATURITY)
```

Comment:
< in one case and > in the other. Shouldn't one of them cover the equality case as well? Otherwise the part will show up both in total available and total unconfirmed, no?
Same would apply to nHeight too I believe?